Bottom: 9ca5abcd2e3eb036c1db03b7b36fa2409de609a1
Top:    4d5bd25fba660f6fe8fd5237559ac065cfdc3fad
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2013-02-11 00:43:07 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
